Shocksupershaman - Copy/src/main/java/com/example/shocksupershaman/shocktask1/UserController.java

```diff
@@ -0,0 +1,45 @@
+package com.example.shocksupershaman.shocktask1;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class UserController {
+
+    @Autowired

```

Comment:
Service variable should be final, we should insert parameters only via constructor, not setters. Add @ReqArgumentsConstructor and remove @Autowired. Spring can handle it without annotations.